README.md

```diff
@@ -1,7 +1,7 @@
 # Calculator
 [![Translation status](https://l10n.elementary.io/widgets/calculator/-/svg-badge.svg)](https://l10n.elementary.io/projects/calculator/?utm_source=widget)
 
-![Screenshot](data/Calculator.png?raw=true)
+![Screenshot](data/screenshot-extended@2x.png?raw=true)
```

Comment:
I used the extended version because the normal @2x screenshot is being displayed at a too large size. Maybe better to use an `<img>` tag with width=(pixel width / 2)?